packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/professional-use/professional-use.controller.js

```diff
@@ -0,0 +1,11 @@
+export default class ProfessionalUse {
+  /* @ngInject */
+  constructor(DedicatedServerFeatureAvailability) {
+    this.DedicatedServerFeatureAvailability = DedicatedServerFeatureAvailability;
+  }
+
+  $onInit() {
+    this.server = this.dedicatedServer.server;
+    this.serviceInfos = this.dedicatedServer.serviceInfos;
```

Comment:
Maybe we can pass those directly as bindings that way we don't need to create a controller